cypress/tests/mocked/parallel-2/cancelObserverDelivery.cy.ts

```diff
@@ -2,22 +2,22 @@ import { toMembersDataApiResponse } from '../../../../client/fixtures/mdapiRespo
 import { observerDelivery } from '../../../../client/fixtures/productBuilder/testProducts';
 import { signInAndAcceptCookies } from '../../../lib/signInAndAcceptCookies';
 
-describe('Cancel home delivery - Sunday', () => {
+describe('Cancel Observer delivery', () => {
```

Comment:
it's still  "home" delivery even if it's observer (as opposed to national delivery)

Reply:
but the Observer is only inside M25 peeps right .. do we need to make that distinction?

Reply:
given the level of context we have, then yes, "observer only delivery" is not available outside M25 at present, but for the test I think it makes sense to be clear.
Although having said that "home delivery" is quite a legacy name